### DISCUSSION TITLE
Wonderful project! Is there an rough roadmap for this?

### DISCUSSION THREAD

**Discussion by antoine12341234** [openblack/openblack#261] [Ideas]
Hi!

I was trying to see if there was any way to reverse-engineer this incredible game and I'm truly glad that I found out that some people are actually already on the task! Thank you so much for bringing hope with this huge workload.

As the title states, I'm just curious to know what kind of "roadmap" this project follows, especially for the following points : 

- With this being an open-source project, would it be feasible to fork the the solution and make a VR version of the game? (note that by "VR version", I'm not referring to playing with the mouse and keyboard but with the headset on, I'm rather referring to using both the headset and the VR controllers)
- After the game is redone with as much accuracy as possible, is it reasonable to think that it would be possible to enhance graphics (resolution, textures, etc.)

I will follow this project closely and if I find some free time, I might as well try to see if I can contribute a bit (if I can manage to understand how the code works, I'm not experienced with C++ as of now).

Thanks a lot for this work and have a great day!

**Comment by bwrsandman** [comment]
I'm doing a bit of work to make porting to the Oculus Quest as little easier by porting to Android #262

**Reply by antoine12341234** [reply]
Ok that's good to know! I would have a question though, it's probably because I'm not knowledgeable about this, but I would like to know what makes the porting to Android "closer" to porting to VR? Again, I'm pretty much at step zero in game development, so my question may seem a little weird.

**Reply by bwrsandman** [reply]
The quest runs android as its operating system so porting to android allows to make sure that:
1. The code compiles on the architecture 64-bit ARM
2. The code compiles with the Android Operating system.

Those are two very basic requirements to get it running. Obviously, using the Oculus API is needed for tracking and rendering as well as changes to our own rendering method by drawing the scene twice from the perspective of each eye. We use bgfx as our rendering layer which has support for Oculus VR.

**Reply by antoine12341234** [reply]
Thank you very much for your work and the detailed explanation!

**Comment by bwrsandman** [comment]
You can have a look at our milestones here https://github.com/openblack/openblack/milestones
